README.md

```diff
@@ -187,6 +187,7 @@ Contributions are welcome. Please take a quick look at the [contribution guideli
  * [crecto-admin](https://github.com/Crecto/crecto-admin) - Admin dashboard for Crecto and your database
  * [micrate](https://github.com/juanedi/micrate) - Database migration tool
  * [migrate](https://github.com/vladfaust/migrate.cr) - A simpler database migration tool with transactions
+ * [migro](https://github.com/aisrael/migro) - A database migration tool written in Crystal
```

Comment:
Please remove "in Crystal" part as it is implied here. Would be nice if you could write about `YML` directly in this title, so the user will see the difference between these tools here, on awesome-crystal page.